docker/testrunner/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
```

Comment:
I'm sorry for such an late answer, but you're completely right, it should be a one liner.